molgenis-dataexplorer/src/test/java/org/molgenis/dataexplorer/DataExplorerControllerTest.java

```diff
@@ -252,9 +256,10 @@ public void testPackageLink()
 	{
 		when(menu.findMenuItemPath(NAVIGATOR)).thenReturn("menu/main/navigation/navigator");
 		when(menuReaderService.getMenu()).thenReturn(menu);
-		Map<String, String> expected = new HashMap<>();
-		expected.put("href", "menu/main/navigation/navigator/packId");
-		expected.put("fullLabel", "parent / pack");
-		assertEquals(controller.getPackageLink(entityTypeId), expected);
+		List<NavigatorLink> expected = new LinkedList();
```

Comment:
Small detail;
Please update to ; List<NavigatorLink> expected = new LinkedList<>();  to fix type warning
Adding the <> has java compiler use the receiving type to set the List impl. 
use sonarLint and inteliJ checks find and fix warnings.